contracts/script/deploy/router/CertVerifierRouterDeployer.s.sol

```diff
@@ -50,13 +47,10 @@ contract CertVerifierRouterDeployer is Script, Test {
 
         // Deploy proxy and initialize in one step
         bytes memory initData =
-            abi.encodeWithSignature("initialize(address,address)", initialOwner, initialCertVerifier);
+            abi.encodeCall(EigenDACertVerifierRouter.initialize, (initialOwner, initialRBNs, initialCertVerifiers));
```

Comment:
Same question here, is this somehow related to the multiple initial addresses change?

Reply:
I realized here that I wanted to make this change because I didn't update this line which caused tests to fail. Using encodeWithSignature does no checks on the types, but encodeCall enforces that the types are the same as what is defined in the contract. This also is a motivation for why I changed the json parsing to typed parsing when possible.

Reply:
Would be good to add if there's no linter for it- there is a TODO in the style guide for adding solidity specific concerns